### PR TITLE
Add cancel anytime tooltip

### PR DESCRIPTION
**Pull request by marialani**
## What are you doing in this PR?
This adds a tooltip feature to provide more detailed information on how a user can go about cancelling their support.

Requirements:
- User taps / clicks on info icon to open tooltip on mobile and desktop respectively
- Clicking outside of the tooltip or using the close button should close it
- Keyboard focus should open tooltip. Can then be closed by navigating away or using the close button inside
- Tooltip and arrow positioning at different breakpoints (above and below reference element)
- Regionalised copy

[**Trello Card**](https://trello.com/c/TA8n5etA/1001-design-and-built-new-tooltip-feature-to-show-more-detail-information-on-cancel-anytime)

## Why are you doing this?
This section receives a fair amount of attention from users.

## Accessibility test checklist
 - [x] [Tested with screen reader](https://webaim.org/articles/voiceover/)
 - [x] [Navigable with keyboard](https://www.accessibility-developer-guide.com/knowledge/keyboard-only/browsing-websites/)
 - [x] [Colour contrast passed](https://www.whocanuse.com/)

## Screenshots
| mobile | mobileMedium | mobileLandscape onwards |
| -- | -- | -- |
| Screenshot 2023-01-31 at 15 52 12 | Screenshot 2023-01-31 at 15 52 30 | Screenshot 2023-01-31 at 15 52 49 |

### CODE REVIEW

**Comment by marialani** [comment]
[Design and build Cancel anytime tooltip](https://trello.com/c/TA8n5etA/1001-design-and-build-cancel-anytime-tooltip)

File: support-frontend/yarn.lock [hunk 1]
```diff
@@ -1661,6 +1661,34 @@
     minimatch "^3.0.4"
     strip-json-comments "^3.1.1"
 
+"@floating-ui/core@^1.2.1":
```
Comment (paul-daniel-dempsey) [inline support-frontend/yarn.lock:1664]: Quite a few new floating dependencies

File: support-frontend/assets/components/tooltip/Tooltip.tsx [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+	countryGroupId: CountryGroupId;
+}): JSX.Element {
+	const [open, setOpen] = useState(false);
+
+	const { x, y, refs, strategy, context } = useFloating({
+		open,
+		onOpenChange: setOpen,
+		placement: 'top',
+		// Make sure the tooltip stays on the screen
+		whileElementsMounted: autoUpdate,
```
Comment (paul-daniel-dempsey) [inline support-frontend/assets/components/tooltip/Tooltip.tsx:98]: I see, anchored even if screen scrolls it stay

File: support-frontend/assets/components/tooltip/Tooltip.tsx [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+		}
+
+		${from.mobileLandscape} {
+			left: 20.5%;
+		}
+	`;
+
+	return (
+		<div css={container}>
+			<div ref={refs.setReference} {...getReferenceProps()}>
```
Comment (paul-daniel-dempsey) [inline support-frontend/assets/components/tooltip/Tooltip.tsx:166]: Prop getter I think described as, linking the close, focus, dismiss, role event listeners with the tooltip context object (containing tooltip display info)

File: support-frontend/yarn.lock (hunk 1 above)
Comment (marialani) [reply support-frontend/yarn.lock:1664]: > Sorry Maria, took me quite a while to grasp how works. Works great so have signed off. Might be worth asking (experienced) Imogen to double check too if you wish? Great job!

I'd say it'd be great if you wouldn't mind checking this out on CODE and testing out the requirements above. If you had already done so I'd say this should be good to go in!

**Pull request merged**